<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="inputEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="addBtn">
         <property name="text">
          <string>添加数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="completeBtn">
         <property name="text">
          <string>添加完成</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>输入的数据：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="inputData">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>762</width>
            <height>73</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="dataLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>排序结果：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="sortResult">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>762</width>
            <height>73</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="orderResult">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>图形化窗口：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="imgDisplay">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>762</width>
            <height>653</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="imgLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextBtn">
         <property name="text">
          <string>下一步</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionexit"/>
    <addaction name="actionfresh"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionfresh">
   <property name="text">
    <string>刷新</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
